eng/Versions.props

```diff
@@ -211,7 +211,7 @@
     <!-- Packages from 2.1/2.2 branches used for site extension build -->
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension21PackageVersion>2.1.1</MicrosoftAspNetCoreAzureAppServicesSiteExtension21PackageVersion>
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension22PackageVersion>2.2.0</MicrosoftAspNetCoreAzureAppServicesSiteExtension22PackageVersion>
-    <MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion>3.1.3</MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion>
+    <MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion>3.1.3-servicing.20163.14</MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion>
```

Comment:
Doubt this is the correct fix since the 3.1.3 version is public. What is really making NuGet.org inaccessible?

Reply:
Is it that the siteextension doesn't stabilize?

Reply:
Second-guessing myself: This is due to the use of this variable to fetch the Microsoft.AspNetCore.AzureAppServices.SiteExtension.3.1.x64 package, not the Microsoft.AspNetCore.AzureAppServices.SiteExtension one. the &hellip;3.1.x64 package is not on NuGet.org. Do I have that right @BrennanConroy ❔

If I'm correct on the reasoning, @wtgodbe and @mmitche aren't we moving away from the dotnet-core feed? This version of the &hellip;3.1.x64 package is available in both https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json and https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet3.1-transport/nuget/v3/index.json

Reply:
Yes we are moving away from the dotnet-core feed. These changes have not yet been checked in yet. Hopefully today.

Reply:
Then dotnet/aspnetcore's 'release/3.1' branch needs to use the https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet2.1-transport/nuget/v3/index.json feed and 'master' and 'release/5.0-preview3' need to use that plus the https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet3.1-transport/nuget/v3/index.json feed? Or, do you mean something else?